german translation for build 125, full android translation (#4703)
Anuken/Mindustry: fastlane/metadata/steam/german/description.txt

[h2]Spielmechanik[/h2]

[list]

> [*] Baue komplexe Produktionsketten aus Förderbändern, um deine Geschütztürme mit Munition zu versorgen und eine große Vielfalt an Ressourcen herzustellen

Reviewer: ```suggestion
[*] Baue komplexe Produktionsketten aus Förderbändern, um deine Geschütztürme mit Munition zu versorgen und erweiterte Ressourcen zum Verbessern deiner Basis herzustellen
```
Auf jeden fall nicht einen nebensatz in aufeinanderfolgenden Punkten wiederholen, vielleicht kann der zweite Teil komplett in den nächsten Punkt, passt hier eigentlich nicht (auch in englicher Version)

[*] Benutze Produktionsblöcke, um eine große Vielfalt an Ressourcen herzustellen
[*] Beschütze deine Basis vor Gegnerwellen

> [*] Spiele mit deinen Freunden in plattformübergreifenden Partien, oder fordere sie zu einem Team-basiertem PvP-Spiel hinaus

Reviewer: ```suggestion
[*] Spiele mit deinen Freunden in plattformübergreifenden Partien, oder fordere sie zu einem Team-basiertem PvP-Match hinaus
```
Match ist passenden, PVP wird als kompetiver Aspekt genannt.

>  plattformübergreifenden Partien

Klingt nicht gut bzw. ungewöhnlich

Author: > plattformübergreifenden Partien

das hatte ich aus der vorherigen übersetzung übernommen, da mir nichts besseres einfiel.

Reviewer: Machmer zu 
'Spiele platformübergreifend mit deinen Freundenä

> [*] Verteile Flüssigkeiten und bekämpfe die ständige Bedrohung von Feuerausbrüchen oder gegnerische Luftangriffe

Reviewer: ```suggestion
[*] Transportiere Flüssigkeiten zu Gebäuten und bekämpfe die ständige Bedrohung von Feuerausbrüchen oder gegnerische Luftangriffe
```
Verteile Flüssigkeiten klingt cursed, habe keine deutsche Beispiele dazu gefunden wie es im Rohrbau so genutzt wird, auch in Jobausschreibungen wird es in der deutschen Übersetzung komplett vermieden durch umschreiben

[*] Nutze deine Produktionsblöcke voll aus, indem du Schmiermittle und Kühlung besorgst

> [*] Stelle eine große Vielfalt an Einheiten her, um deine Basis zu beschützen und zu verwalten

Reviewer: ```suggestion
[*] Stelle eine Vielfalt an Einheiten her, welche je nach Typ Angriffe und Verteidigung oder Abläufe in der Basis unterstützen können
```

[list]
[*] Erobere den Planeten Serpulo, indem du über 250 prozedural generierte Sektoren und 16 handgemachte Karten eroberst
[*] Erorbere dein Gebiet und stelle Fabriken auf, um Ressources zu besorgen, während du in anderen Sektron weitermachst
[*] Verteidige deine Sektoren vor regelmäßigen Invasionen
[*] Koordiniere Ressourcesverteilung zwischen Sektoren mit Launchpads
[*] Erforsche neue Blöcke, um deinen Fortschitt foranzutreiben
[*] Schließe Missionen zusammen mit deinen Freunden ab
[*] 130+ Blöcke
[*] 33 verschiedene Einheiten
[*] über 50 Erfolge
[/list]

[h2]Spielmodi[/h2]

[list]

> [*] [b]Überleben[/b]: Baue Geschütztürme, um deine Basis zu beschützen. Überlebe so lange wie möglich und starte deine Ressourcen, um mit ihnen zu forschen. Bereite deine Basis auf gegnerische Bossangriffe vor.

Reviewer: ```suggestion
[*] [b]Überleben[/b]: Baue Geschütztürme, um deine Basis im Tower-Defense Style zu beschützen. Überlebe so lange wie möglich und starte deine Ressourcen, um mit ihnen zu forschen. Bereite deine Basis auf gegnerische Bossangriffe vor.
```
Falls der Kunde den Factorio-Vibe nicht bekommt, nochmal Tower-Defense erwähnt

[*] [b]Angriff[/b]: Baue Fabriken für Einheiten, die den gegnerischen Kern zerstören und währenddessen deinen beschützen. Erstelle eine Mischung aus defensive und offensive Einheiten, um dich zu unterstüzen. Aktiviere wahlweise eine KI, sodass der Gegner auch bauen kann.

> [*] [b]PvP[/b]: Kämpfe mit bis zu vier Teams um den Sieg und zerstöre die gegnerischen Kerne. Baue Einheiten oder greife die Gegner direkt an.

Reviewer: ```suggestion
[*] [b]PvP[/b]: Kämpfe in bis zu vier Teams um den Sieg und zerstöre die gegnerischen Kerne. Erschaffe Einheiten oder greife die Gegner direkt an.
```
Du bist einer der 4 Teams also in, erschaffe passt glaube eher

[*] [b]Sandkasten[/b]: Spiele mit unendlichen Ressources und ohne Gegnergefahr. Benutze besondere Ressourcenquellen um Sachen auszuprobieren, oder erstelle Gegner auf Anfrage.
[/list]

[img]{STEAM_APP_IMAGE}/extras/2020-11-30_10-46-02.gif[/img]

[h2]Mehrspielermodus[/h2]

[list]
[*] 16 eingebaute Karten für eigene Spiele
[*] Spielt gemeinsam, gegeneinander oder im Sandkasten
[*] Trete einem öffentlichen Server bei oder eröffne ein Spiel für deine Freunde
[*] Anpassbare Spielregeln: Verändere Blockkosten, Gegnerstärke, Anfangsitems, Wellen und mehr!
[*] Kombiniere PvP und PvE (player versus enemies)
[/list]

[h2]Karteneditor[/h2]

[list]
[*] Male Terrain
[*] Editiere und sehe eine Vorschau deiner Karte im Spiel
[*] Passe die Werkzeuge nach deinen Vorstellungen an
[*] Nützliches Generationssystem für Karten mit vielen verschiedenen Filtern für das Bearbeiten vom Terrain
[*] Verschönere die Landschaft mit Rauschen, Verzerrungen, Glätte, Erosion, Symmetrie, Erzgeneration und Zufallsgenerierung
[*] Lasse Erze, Flüsse und andere Ressourcen automatisch generieren
[*] Konfiguriere die Gegnerwellen
[*] Teile deine Karten im Steam-Workshop

> [*] Verändere die Karten-Regeln

Reviewer: ```suggestion
[*] Gib der Karte besondere Spielregeln
```
Nicht perfekt, aber hier soll erwähnt werden das Spielregeln beim spielen der Karte angepasst werden können.
Karten-Regeln klingt nicht verständlich, vorallem beim betrachten vorheriger Elemente
